Hazelcast.Net/Hazelcast.Config/ClientNetworkConfig.cs

```diff
@@ -76,12 +79,21 @@ public class ClientNetworkConfig
 
         private SSLConfig _sslConfig = new SSLConfig();
 
+        /// <summary>
+        /// Adds given addresses to candidate address list that client will use to establish initial connection
```

Comment:
...addresses to the list of candidate addresses ... connection. (standardizing the expression of this list).